### ISSUE TITLE
Meta viewport tag is not accessible

### ISSUE THREAD

**Issue by heikki1200** [quasarframework/quasar#17031]
### What happened?

Meta viewport tags's scaling is disabled and causes issue with accessibility.
[https://www.w3.org/TR/WCAG21/#resize-text](https://www.w3.org/TR/WCAG21/#resize-text)

Users should be able to zoom at least 200%.

### What did you expect to happen?

Zooming should be enabled for people with bad eyesights. This is critical feature and was caught when doing accessibility auditing.

### Reproduction URL

[https://stackblitz.com/edit/quasarframework-cuw3dv](https://stackblitz.com/edit/quasarframework-cuw3dv
)

### How to reproduce?

Run accessibility tests with WCAG 2.1 AA guidelines.

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Accessibility [a11y] (quasar)

**Comment by rstoenescu** [comment]
Fixed in cf5514e79 — the meta viewport tag in web modes no longer disables zooming (user-scalable/maximum-scale removed), satisfying WCAG 2.1 AA 1.4.4. Will ship with the next release. Thank you for the report!

**Comment by rstoenescu** [comment]
One further note: spawned apps will need a manual update of their viewport in index.html. And for completeness sake, upcoming UI v2.25.0 will ensure everything works as expected.
